solidity_contracts/src/CairoPrecompiles/CairoCounterCaller.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./CairoLib.sol";
+
+contract CairoCounterCaller  {
+    /// @dev The cairo contract to call
+    uint256 cairoCounterAddress;
+
+    /// @dev The cairo function selector to call - `inc`
+    uint256 constant FUNCTION_SELECTOR_INC = uint256(keccak256("inc")) % 2**250;
+
+    /// @dev The cairo function selector to call - `set_counter`
+    uint256 constant FUNCTION_SELECTOR_SET_COUNTER = uint256(keccak256("set_counter")) % 2**250;
+
+    /// @dev The cairo function selector to call - `get`
+    uint256 constant FUNCTION_SELECTOR_GET = uint256(keccak256("get")) % 2**250;
```

Comment:
let's do some overloading of the lib functions to remove the need to precompute the selectors (ofc would cost more at runtime — a keccak — but may be easier to use